app/src/org/commcare/android/database/global/models/ApplicationRecord.java

```diff
@@ -3,35 +3,58 @@
  */
 package org.commcare.android.database.global.models;
 
+import org.commcare.android.database.SqlStorage;
+import org.commcare.android.database.app.DatabaseAppOpenHelper;
+import org.commcare.android.database.app.models.UserKeyRecord;
+import org.commcare.android.database.user.CommCareUserOpenHelper;
 import org.commcare.android.storage.framework.MetaField;
 import org.commcare.android.storage.framework.Persisted;
 import org.commcare.android.storage.framework.Persisting;
 import org.commcare.android.storage.framework.Table;
+import org.commcare.dalvik.application.CommCareApp;
+import org.commcare.dalvik.application.CommCareApplication;
+
+import android.content.Context;
 
 /**
- * An Application Record tracks an indvidiual CommCare app on the current
+ * An Application Record tracks an individual CommCare app on the current
  * install.
  * 
  * @author ctsims
+ * @author amstone
  *
  */
+
 @Table("app_record")
 public class ApplicationRecord extends Persisted {
-    public static final String META_STATUS = "status";
+    public static final String STORAGE_KEY = "app_record";

```

Comment:
Comment what this is, maybe something like:
"Name of database that stores application records"